src/Tests/HttpServerIntegrationTests.cs

```diff
@@ -74,7 +74,7 @@ public async Task CompileAndSimulateViaApi()
                 // TODO: Use constant strings for these http requests. Port and paths.
 
                 // Compile a snippet with the server and get back a list of now executable operations.
-                var compileResult = await "http://localhost:8888/api"
+                var compileResult = await "http://localhost:8008/api"
```

Comment:
Ah, this was because every bloody time I ran tests locally they failed due to conflicting with a Jupyter Notebook session I had open in the background. Making the tests uses a port that's less likely to conflict with Jupyter improves DX significantly.